### PR TITLE
Adds a fixture for what the entire DSL JSON looks like for BBS

### PR DESCRIPTION
**Pull request by orta**
Thanks to @azz's solid tests this was pretty trivial  👍 

/cc @f-meloni 

merge on green

### CODE REVIEW

**Comment by KITSFrancoMeloni** [comment]
Thank you :)

**Comment by f-meloni** [comment]
merge on green

File: source/platforms/bitbucket_server/_tests/_bitbucket_server_git.test.ts [hunk 1]
```diff
@@ -135,4 +136,12 @@ describe("the dangerfile gitDSL - BitBucket Server", async () => {
     const promise = bbs.findTypeOfLine(gitDSL, 0, "jest.eslint.config.js")
     await expect(promise).rejects
   })
+
+  it("writes a JSON DSL fixture", async () => {
+    const fakeSource = new FakeCI({})
+    const dataSent = await jsonDSLGenerator(bbs, fakeSource, {} as any)
+    dataSent.settings.github.accessToken = "12345"
+
+    writeFileSync(pathJoin(fixtures, "bbs-dsl-input.json"), JSON.stringify(dataSent, null, "  "), "utf8")
```
Comment (hanneskaeufler) [inline source/platforms/bitbucket_server/_tests/_bitbucket_server_git.test.ts]: A bit late to the game now, but does this file have any usage yet? This test produces local (looks like formatting only) changes for me after a test run and this test doesn't seem to have any verifications? Can we remove it?

Comment (orta) [reply source/platforms/bitbucket_server/_tests/_bitbucket_server_git.test.ts]: I've fixed that in #778 

But yeah, it does - it's used in Danger Swift and Kotlin, it's means to act as a reference for other language runners

**Pull request merged**